labelbox/schema/model_run.py

```diff
@@ -122,6 +125,34 @@ def _wait_until_done(self, status_fn, timeout_seconds=120, sleep_time=5):
                 )
             time.sleep(sleep_time)
 
+    def upsert_predictions_and_send_to_project(
+        self,
+        name: str,
+        predictions: LabelList,
```

Comment:
Unfortunately we can't count on this type being a LabeList. It could be ndjson, a url that points to ndjson, or a LabeList.